soc/arm/microchip_mec/common/spigen/mec_spi_gen.py

```diff
@@ -0,0 +1,476 @@
+# -*- coding: utf-8 -*-
```

Comment:
Add add python shebang? (see the other python scripts in the repository)

And I think you don't need the utf-8 tag with python3 so that should be removed.

Reply:
updated